### ISSUE TITLE
"File wuaueng.dll is under the minimum supported version" error during installation

### ISSUE THREAD

**Issue by samotloruser** [zeffy/wufuc#99]
error during installation of the wufuc v0.7.1.81: "File WUAUENG.dll is under the minimum supported version" on win7x64SP1 Russian

**Comment by samotloruser** [comment]
WUAUENG.dll file version 7.6.7600.320
SHA1 6acea7a16c5cd94764c7f7d992fda371924367e6 *wuaueng.dll

**Comment by zeffy** [comment]
Hi, that just means your PC doesn't have any updates installed that contain the CPU check that wufuc disables. You must first run Windows Update to get the latest quality rollup update, and then install wufuc afterwards, and it should work correctly.

**Comment by rautamiekka** [comment]
It's quite an ambiguous message, so couldn't it be changed to something better ? I'm thinking something like
```
WUAUENG.dll is older than the version when the processor check was introduced.
Run Windows Update until the "Unsupported Hardware" window shows up, then try again.
```

**Comment by samotloruser** [comment]
> Hi, that just means your PC doesn't have any updates installed that contain the CPU check that wufuc disables. You must first run Windows Update to get the latest quality rollup update, and then install wufuc afterwards, and it should work correctly.

yes, you are right, thanks. Just one moment. After installing wufuc I ran the update search again and there were no longer any optional updates at all although it should be about 60

**Comment by zeffy** [comment]
@rautamiekka Yes, I have plans to clarify this error message in the next version.

@samotloruser I'm not really sure what's causing that. My patch doesn't modify with anything other than the flag that decides if your processor is supported, not anything to do with what updates are actually available. Are you positive there were ~60 updates before you installed wufuc?

**Comment by samotloruser** [comment]
> Are you positive there were ~60 updates before you installed wufuc?
yes it was. After a few hours, I started searching for updates again and 65 optional updates were found ))
I do not know why.  Thank`s for your work. 
It would be good to add another way of donate without paypol.

**Comment by zeffy** [comment]
@samotloruser sorry, I just now saw your last comment!

>It would be good to add another way of donate without paypol.

I'm certainly open to suggestions, do you know of any donation platforms that don't require paypal?

**Comment by zeffy** [comment]
Regarding this issue, I've made changes to make the error message to make the meaning more clear. The change will be included in v0.7.2 when I release it.
